spec/services/work_combined_audio_creator_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
```

Comment:
Add a couple tests for fingerprint, make sure that if the members change in a way that should change fingerprint (add or remove member, change position), it does change fingerprint.

Reply:
Done. Took forever but I don't regret it.